install-neon-lib should depend on libsvn_delta's installation

install-neon-lib needs libsvn_delta-1.la which will be regenerated
during libsvn_delta-1.la's installation, if libsvn_delta-1.la is
in regenerating and at the same time install-neon-lib links it, the
error willl happen.

Let install-neon-lib run after libsvn_delta-1.la is installed will fix
the problem.

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 build-outputs.mk |    2 +-
 build.conf       |    2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/build-outputs.mk b/build-outputs.mk
--- a/build-outputs.mk
+++ b/build-outputs.mk
@@ -979,7 +979,7 @@ install-locale: subversion/po/de.mo subversion/po/es.mo subversion/po/fr.mo subv
 	$(MKDIR) $(DESTDIR)$(localedir)/zh_TW/LC_MESSAGES
 	cd subversion/po ; $(INSTALL_LOCALE) zh_TW.mo $(DESTDIR)$(localedir)/zh_TW/LC_MESSAGES/$(PACKAGE_NAME).mo
 
-install-neon-lib: subversion/libsvn_ra_neon/libsvn_ra_neon-1.la
+install-neon-lib: subversion/libsvn_ra_neon/libsvn_ra_neon-1.la $(SVN_FS_LIB_INSTALL_DEPS)
 	$(MKDIR) $(DESTDIR)$(neon_libdir)
 	cd subversion/libsvn_ra_neon ; $(INSTALL_NEON_LIB) libsvn_ra_neon-1.la $(DESTDIR)$(neon_libdir)/libsvn_ra_neon-1.la
 
diff --git a/build.conf b/build.conf
--- a/build.conf
+++ b/build.conf
@@ -272,6 +272,8 @@ type = ra-module
 path = subversion/libsvn_ra_neon
 install = neon-lib
 libs = libsvn_delta libsvn_subr aprutil apriconv apr neon
+# conditionally add more dependencies
+add-install-deps = $(SVN_FS_LIB_INSTALL_DEPS)
 msvc-static = yes
 
 # Accessing repositories via DAV through serf
-- 
1.7.10.4

